modules/ROOT/pages/ios_app/overview.adoc

```diff
@@ -0,0 +1,31 @@
+= Overview
+:toc: right
+:description: Building and distributing your branded iOS app involves a larger number of interdependent steps. The process is detailed in this chapter over several pages.
+
+== Introduction
+
+{description} Follow these instructions exactly in order to have a branded iOS app that you can distribute to your users.
```

Comment:
Maybe add a note that iOS docs are partially outdated, and customers should get in touch with oC Support team for more information.

Pointers:
- iOS ownBrander now in http://portal.owncloud.com
- Current signing instructions: https://github.com/owncloud/ios-app/tree/master/enterprise/resign

@ValentinKruegl is working on the update.

Reply:
@michaelstingl as written in the PR description, I do **NOT** want to do all the process cleanups here as this will explode the PR. Let us merge this one as it reflects the current process. In a second step we can do of course all the (necessary) process cleanups and tunings/improvements. This is much easier when doing on the updated base.